content/learn/case-studies/cnbc.md

```diff
@@ -0,0 +1,50 @@
+---
+$order: 14
+$title: CNBC
+$date: May 22, 2017
+$category: Publishers
+$parent: /content/learn/case-studies.html
+class: case-study post-detail
+
+components:
+  - social-share
+
+headline: CNBC sees mobile speed and engagement gains with AMP
+thumb: "case-studies/cnbc_logo_thumb.jpg"
+results:
+    - stat: 22%
+      description: Increase in mobile search users returning to CNBC.com in 7 days
+    - stat: 387%
+      description: Increase in mobile page load time
+
+logo:
+  src: "case-studies/cnbc_logo_thumb.png"
+  width: 200
+  height: 35
+download: "case-studies/cnbc.pdf"
+
+---
+
+<div class="img-right">
+    <amp-img width="271" height="539" layout="responsive" src="/static/img/case-studies/cnbc1.png"></amp-img>
+</div>
+
+CNBC works hard to ensure that their users have access to the content they need, wherever they need it. With mobile traffic now accounting for roughly 50% of CNBC's total traffic, it has become a strategic priority to provide a fast loading, less frustrating user experience to this expanding user base. “Improving performance for all our pages is a top priority. As we see our traffic from mobile devices increasing, it’s become paramount we provide the fastest experience we can for those users,” said Viktor Cea, Director of Content and Design. The AMP format offers an enhanced experience for users, with its promise of near-instant page load times. 
+
+## Solution
+
+The CNBC team took a multi-phased rollout approach to deploying AMP across text articles, releasing a minimum viable product (MVP) in late-June of 2016. 
+ 
+The team was able to quickly ramp up the project since the AMP format leverages many existing web standards, allowing CNBC to implement their standard analytics tools to measure usage. The phased rollout also afforded them the opportunity to optimize their AMP code markup with the help of the AMP validation tools. 
+ 
+Within a few months of the initial launch, AMP traffic crossed into the double digits as a percentage of mobile web traffic, which meant it was time to monetize and re-circulate this new traffic stream. CNBC followed up the MVP release by implementing ads via the standardized < amp-ad > tag. They were also able to work with a third party vendor to rapidly deploy lightweight, AMP-specific code for internal content re-circulation for users.
```

Comment:
I think you should put the < amp-ad > tag in code style, start with a back-tick<amp-ad>back-tick, like so:

```
`<amp-ad>`
```

It will appear like this: ....standardized `<amp-ad>`  tag.